### ISSUE TITLE
Adjust the position & size of various elements in edit view

### ISSUE THREAD

**Issue by jakekall** [ohtuprojekti-eficode/Amandus#222]
Currently:
- When a file is opened, the name of the file is touching the sidebar
    - Should be moved slightly to the right
- The save button is touching the editor and the sidebar
    - Should be moved slightly down and right
- When user goes to edit view and selects a file, the area above the editor expands to fit the name of the file causing the editor to shift down
    - The area above the editor should be big enough for the filename right away
- The area below the editor is probably too big, atleast on larger screen sizes and especially after #220 is done
    - Make the bottom area smaller
    - A more solid solution might be to have a separate bottom bar component with the save button in it but that would require refactoring on how the saving is done. If you want to implement it this way, a new issue should probably be created for this part
